src/client/actions/gaEvents/index.tsx

```diff
@@ -0,0 +1,37 @@
+import ReactGA from 'react-ga'
+import { get } from '../../util/requests'
+
+// Checks if reactga has initalised
+let hasInit = false
+
+const getGaId = async () => {
+    await get('/api/ga').then(async (response) => {
+      if(response.ok) {
+        await response.text().then((gaid) => {
+            ReactGA.initialize(gaid)
+            hasInit = true
+        })
+      }
+    })
+  }
+// Records clicks - customise action and label name to record multiple uniques per session
```

Comment:
Given that these will be extensively used through the codebase, it might be helpful to have these as jsdoc so that they can turn up in the editor